plugins/lsp/language-servers/default.nix

```diff
@@ -739,27 +171,48 @@ let
           };
         };
       };
-    }
-    {
-      name = "yamlls";
-      description = "yamlls for YAML";
-      package = "yaml-language-server";
+    };
+    yamlls = {
       settings = cfg: { yaml = cfg; };
-    }
-    {
-      name = "zls";
-      description = "zls for Zig";
-    }
-  ];
-  renamedServers = {
-    tsserver = "ts-ls";
+    };
   };
+
+  lspPackages = import ../lsp-packages.nix;
+
+  getLspPackage =
```

Comment:
I find it a bit confusing that you can activate an lsp that doesn't exist eg angularls and don't get any warning that there is no package for it. Don't get me wrong, I love it, that it can now be easily extended I just want to know when package is null and I need to manually set it :)

Reply:
see https://github.com/NuschtOS/nvim.nix/pull/61 , specifically https://github.com/NuschtOS/nvim.nix/commit/938a5c576c67ae7651094974c35f8620f3611916

Reply:
We have that behaviour for none-ls, so I thought that the lsp servers were the same, it is indeed much better to require an explicit null to acknowlege that the server must be installed by other means

Reply:
OTTOMH, the other places where this is possible (none-ls & efmls) have no-default package options in cases where we don't have a known package.

This forces the user to set the option explicitly or they will get "option accessed but not defined" assertion errors.